pom.xml

```diff
@@ -5,17 +5,18 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.2.2.RELEASE</version>
+		<version>2.6.6</version>
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
 	<groupId>org.icgc.argo</groupId>
 	<artifactId>workflow-management-scheduler</artifactId>
-	<version>1.3.0</version>
+	<version>1.4.1-SNAPSHOT</version>
```

Comment:
Merging into master, lets get rid of the SNAPSHOT version. we should be fine with `1.4.1`